### PR TITLE
February Newsletter

### PR DESCRIPTION
**Pull request by EdwinB12**

### CODE REVIEW

**Review by Robadob — APPROVED** [review]
Given it a quick read through whilst waiting for a meeting to start. Looks good to me, nothing I'd suggest changing (which is unusual, so not sure if I skimmed it too fast).

File: _newsletters/2023-02-24-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+* [Apply for the current round of Carpentries Maintainer Onboarding!](https://carpentries.org/blog/2023/02/Apply-for-the-current-round-of-Carpentries-Maintainer-Onboarding/) - Applications due 1st March. - The Carpentries is preparing to onboard the next cohort of Maintainers. Currently we have 109 Maintainers serving across 56 core lessons. To keep the community healthy, we need to plan for offboarding people who want to step down from their current positions and finding new people to step into the role.
+* [Official MathWorks MATLAB kernel for Jupyter released](https://blogs.mathworks.com/matlab/2023/01/30/official-mathworks-matlab-kernel-for-jupyter-released/)
+* [Launching Data Carpentry: Image Processing with Python](https://carpentries.org/blog/2023/01/dc-image-processing-stable-release/) - Announcing the stable release of a new curriculum teaching image processing skills. Sign up to host a workshop today!
+* [Tutorials on Citation File Format](https://www.youtube.com/watch?v=zcgLIT5Qd4M) - In this tutorial, the cffinit v2.2.0 tool is used to create a CITATION.cff file.
```
Comment (ns-rse) [inline _newsletters/2023-02-24-newsletter.md]:
```suggestion
* [Tutorials on Citation File Format](https://www.youtube.com/watch?v=zcgLIT5Qd4M) - In this tutorial, the [cffinit v2.2.0 tool](https://citation-file-format.github.io/cff-initializer-javascript/#/) is used to create a CITATION.cff file.
```

File: _newsletters/2023-02-24-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+#### Research IT Training
+
+Research IT courses will be adopting a hybrid approach from March. The team will be providing their courses both online and in person for the first time since March 2020. The team provides a place for beginners or advanced users to expand their knowledge of HPC and different programming languages. The courses are part of the Doctoral Development Programme. For more information please visit our training registration web page (via VPN): [crs.shef.ac.uk](https://crs.shef.ac.uk).
+
+
+#### RSE Team: Git & GitHub through GitKraken workshop - from Zero to Hero!
+
+This is an introductory course, teaching the git and GitHub skills required to manage research code over it’s development lifecycle. These skills are essential for collaborative research teams.
+
```
Comment (ns-rse) [inline _newsletters/2023-02-24-newsletter.md]:
```suggestion
* [Introduction to Conda for (Data) Scientists 2023-03-03](https://rse.shef.ac.uk/training/workshop/2023-03-03-intro-to-conda)
```

**Review by ns-rse — CHANGES_REQUESTED** [review]
Looks fine to me, have suggested a link to the CFFInit site and including next weeks Conda course.

**Comment by EdwinB12** [comment]
Thanks @ns-rse - Made both of those suggested changes. Just need your approval before merging :)

**Comment by EdwinB12** [comment]
Thanks for your reviews @ns-rse and @Robadob

**Pull request merged**